rtk_gps_locosys_r1.md

```diff
@@ -0,0 +1,95 @@
+# Freefly Systems RTK GPS
```

Comment:
There is something very wrong with this. There is already a document with this name, and the correct title and value: 
http://docs.px4.io/main/en/gps_compass/rtk_gps_locosys_r1.html

I think you probably started off an old mainline. YOu probably need to rebase against the mainline then start again.

Reply:
Ah, I get it, you have submitted this in error.